### DISCUSSION TITLE
[IntelliSense] Support Rust Dioxus

### DISCUSSION THREAD

**Discussion by John0x** [tailwindlabs/tailwindcss#7073] [Ideas]
Hey, would it be possible to support Rust with the Dioxus framework?
https://github.com/DioxusLabs/dioxus

They use a custom jsx like syntax which doesn't seem to work this extension

**Comment by balenamiaa** [comment]
```js
"tailwindCSS.experimental.classRegex": [
    "class\\s*:\\s*\"([^\"]*)",
],
```

**Comment by mkmik** [comment]
what else do I need to setup in order for tailwindcss to even look at my *.rs files?

**Reply by kzhui125** [reply]
```json
{
    "editor.quickSuggestions": {
        "other": "on",
        "comments": "on",
        "strings": true
    },
    "tailwindCSS.experimental.classRegex": [
        "class\\s*:\\s*\"([^\"]*)",
    ],
}
```

**Comment by VesperQuartz** [comment]
for neovim what do i do?

**Reply by rafaelsmgomes** [reply]
I use this plugin:
https://github.com/tamago324/nlsp-settings.nvim

It gives you the ability to add lsp-settings to the local project.
So, you run `LspSettings local tailwindcss` and it will create a folder called .nlsp-settings/tailwindcss.json

Then you can add the configuration above.
But also add "rust" as a language.
```yaml
{
  "tailwindCSS.includeLanguages": {
    "rust": "html"
  },
  "tailwindCSS.experimental.classRegex": [
    "class\\s*:\\s*\"([^\"]*)"
  ]
}
```

If you set it up correctly, you should also get intellisense for the configuration. 
This is super useful specially for tailwindcss, since you can have local configuration for projects.
No more adding tailwindcss to every rust project!

Hope this helps.

PS: nlsp-settings used to be a recommended way of adding local configuration to lsp servers. I don't know why it is no longer mentioned in the Wiki for nvim-lsp
